plutus-pab/src/Plutus/PAB/Core.hs

```diff
@@ -67,7 +67,6 @@ module Plutus.PAB.Core
     , activeContracts
     , finalResult
     , waitUntilFinished
-    , blockchainEnv
```

Comment:
Why remove this one?

Reply:
It's `PABEnvironment`'s field that is being exported as `PABEnvironment(..)`.